src/main/java/mekanism/common/util/StackUtils.java

```diff
@@ -11,8 +11,8 @@
 
     //ignores count
     public static boolean equalsWildcard(ItemStack wild, ItemStack check) {
-        if (wild.isEmpty() || check.isEmpty()) {
-            return check == wild;
+        if (wild.isEmpty() && check.isEmpty()) {
```

Comment:
I am not quite sure this is a correct change (or at least fully correct), as if wild is empty we want it to return false if check is not empty, however I think the below checks will now happen. I do agree the previous `check == wild` is not the proper way to check to make sure they are both empty, however I think the new way misses a false branch.

Reply:
it probably would have always returned false here anyway. If only one is empty the checks below will fail, because its not equal anyway

Reply:
I agree that the check was probably returning false here when it shouldn't have been, but doesn't `isEmpty()` also return true if the stack size is 0, so technically the other data could still match if something is giving us a malformed ItemStack?

Reply:
```
public Item getItem()
    {
        return this.isEmpty || this.delegate == null ? Items.AIR : this.delegate.get();
    }
```
nope

Reply:
Forgot to look at things if they return different values if it is empty.